helm/prometheus-rules/templates/recording-rules/gs-managed-app-deployment-status.rules.yml

```diff
@@ -9,21 +9,36 @@ spec:
   groups:
   - name: gs-managed-app-deployments.recording
     rules:
-    - expr: label_replace(
-              kube_deployment_status_replicas_available *
-              on (namespace, deployment) group_left(label_app_kubernetes_io_name)
-              kube_deployment_labels{label_giantswarm_io_service_type="managed"},
-            "managed_app", "$1", "label_app_kubernetes_io_name", "(.*)" )
+    - expr: |
+        label_replace(
+            kube_deployment_status_replicas_available
+          * on (pod, namespace, deployment) group_left (label_app_kubernetes_io_name)
```

Comment:
Add the pod label in the grouping because we multiple KSM replicas, we get the same metric with the only differences being the pod and node names